### ISSUE TITLE
Avira free anti virus version 14.0.7.468: NVDA announces Unknown for a control a tab opened through the Extras submenu > Detections List > Help button > Ctrl + Tab once

### ISSUE THREAD

**Issue by nvaccessAuto** [nvaccess/nvda#4806]
**Reported by bhavyashah on 2015-01-14 13:20**
Steps to reproduce
1 Download, install and open Avira free anti virus version 14.0.7.468.
2 Go to Extras submenu > Detections list > Help button.
3 Press Ctrl + Tab once to go the next tab.
4 Press tab once from the ‘Type in the keyword to find’ edit box.
Exppected Result : NVDA will announce something.
Actual Result : NVDA says ‘unknown’, and while pressing NVDA + TAB on the same control, NVDA again reports ‘unknown focused’.
I’ll stop commenting on Narrator’s behaviour, since its either that I am unable to use Narrator efficiently enough, or Narrator is too poor a screen reader worth mentioning/comparing with NVDA.Test Environment : Windows 7 Ultimate 32 bit
Avira free anti virus version 14.0.7.468
NVDA version 2014.4 stable
Update: While playing about a bit more, I discovered that you can reach the inaccessible ‘unknown’ control directly by going into the Help submenu > Overview.

**Comment by leonardder** [comment]
@bhavyashah: Could you still reproduce this with Avira?

**Comment by bhavyashah** [comment]
@leonardder Unfortunately, I do not use Avira anymore. Hence, I will be unable to provide a status update on this issue. However, if there are a bunch of Avira specific issues on the tracker (most of which would probably have been filed by me), I could probably seek assistance from an existing Avira user in testing these by asking on the NVDA Users e-group.

**Comment by Adriani90** [comment]
As I said in previous issue, the interface has been changed significantly and the menu structure is totally different. Actually NVDA works in browse mode in Avira now. So I close this issue as invalid because of interface change. Please open a new one with current steps to reproduce. Thanks.
